entries/s/swisscom.ch.json

```
@@ -1,17 +1,15 @@
 {
   "Swisscom": {
     "domain": "swisscom.ch",
-    "url": "https://www.swisscom.ch/",
-    "contact": {
-      "facebook": "Swisscom",
-      "twitter": "Swisscom_de",
-      "language": "de"
-    },
+    "tfa": [
+      "sms"
+    ],
```

```suggestion
    "tfa": [
      "sms",
      "custom-software"
    ],
    "custom-software": [
      "Mobile ID"
    ],
```
According to the documentation page: `receive an SMS code to enter after you log in, or confirm access using Mobile ID. `

---

How did I miss that? :raised_eyebrow: